lib/src/components/NxCollapsibleItems/types.ts

```diff
@@ -31,6 +32,10 @@ export const propTypes: PropTypes.ValidationMap<Props> = {
     PropTypes.arrayOf(PropTypes.node),
     PropTypes.node
   ]),
+  actionContent: PropTypes.oneOfType([
+    PropTypes.arrayOf(PropTypes.node),
+    PropTypes.node
+  ]),
```

Comment:
I'm pretty sure that `PropTypes.node` can match an array anyway, it's basically equivalent to `ReactNode`.  But anyway, just like the `ReactNode` above should be `ReactElement`, this should be `PropTypes.element`.

Reply:
> I'm pretty sure that `PropTypes.node` can match an array anyway, it's basically equivalent to `ReactNode`. But anyway, just like the `ReactNode` above should be `ReactElement`, this should be `PropTypes.element`.

Thanks, I updated that here: ddfde84270f8f18a0e362d2d12f8dc9f40f0d774